products/platform/product-slo-tools/README.md

```diff
@@ -0,0 +1,157 @@
+
+**SLAs:** service level agreements. These are targets that a service team intends its system to hit, with consequences if those targets are not hit.
+
+Read more:
+
+- [SLO Fundamentals](https://cloud.google.com/blog/products/gcp/sre-fundamentals-slis-slas-and-slos)
+- [Defining SLOs](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/policies-work-norms/how-to-write-your-slos.md)
+- [Google's SRE Approach](https://landing.google.com/sre/workbook/toc/)
+
+## Problem Statement
+
+VFS teams don't have a consistent way to drive product decisions based on performance metrics. In order to
+improve self-service of teams building on VSP, we need to have a clear process in place for the entire SLO lifecycle.
+
+This includes:
+
+- defining service-level objectives for products based on user experience
+- monitoring and alerting based on performance of SLOs
+- finding support for this process
+
+#### Requirements
+
+This is what needs to happen with any implemented solution to achieve our desired outcome.
+
+- define the list of products and the responsible team(s)
+- define what documentation will be needed and where
+  - provide templates & baseline definitions for SLO configuration
+  - define the enforcement policy for SLO performance on VSP ( is this a checkbox in collab cycle? Or a metric-based PR failure?)
+  - document VFS playbook for when the SLO suffers
+- outline initial SLO touchpoint meeting with VFS team & VSP tools team
+
+
+##### Acceptance Criteria
+
+- SLOs are present and all stakeholders for the product approve
+- People responsible for product agree it is possible to meet SLOs as defined
+- Organization commits to using error budget for decision making/prioritizing work effort
+- Process is in place for refining the SLO over time
+
+
+## Proposed Solution
+
+- Clearly delineate and define what makes a product on VSP, starting with the `Source-App-Name` header/`manifest.json`
+- Assign/identify appropriate stakeholders & VFS team for each application, updating `product/` docs as needed
+- Work with VFS teams to configure sensible initial SLOs for each product -
+    [reference](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/policies-work-norms/how-to-write-your-slos.md)
+- Define a centralized configuration template (JSON? YAML?) that can be used in monitoring, reporting, scripts, etc.
+- Leverage this tool-agnostic configuration to centralize the source of truth for all product SLOs
+- All SLO performance config could be in one place rather than disparate places (PagerDuty, Prometheus Alerts, Grafana,
+    etc)
+
+Proposed sample configuration, using search product:
+
+  ```yaml
+    ---
+    search:
+      action: 'index' # which action?
+      controller: 'v0/search' # which endpoint?
```

Comment:
Does SLO/SLA apply also to backend services? (like EVSS for example) - Or is that outside the scope of what we want to accomplish?

Reply:
It'll be outside of the scope for this juncture, we will likely want to look at ways that we can normalize external and on-platform if possible.